Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -400,6 +400,10 @@ YUI.add('ez-contenteditviewservice', function (Y) {
         _redirectToAttribute: function (attr) {
             var attrRedirectionUrl = this.get(attr);
 
+            this.set('publishRedirectionUrl');
```

Comment:
> the default value is undefined so I'm setting the default value.

which is indeed true but this info is completely absent when you read the code you are adding (or the existing code, while doing that the attribute doc block should be fixed as well). So it's clear for you now, not for me nor anybody else (even you) in a near future.

> There's no need to keep them after redirecting a user.

yeah but you do it before redirecting not after.

> Keeping it after user redirection makes the app behaviour unpredictable when someone wants to update it and redirect user somewhere else.

that's right but you are assuming here that using one of this attribute is the only way to navigate away from the current route associated with the view service. If you navigate differently (for instance for whatever reason, the editor is using a simple link available in the page), the attribute won't be reseted because you tie the reset logic with the navigation logic with one of those attributes.

> If it still doesn't convinvce you, then what you propose to do in this case?

the problem is not convincing me, the problem is fixing the issue correctly both in terms of code organization/quality (yes create a separate method to reset the redirection attributes with a good name and a few lines of description is a good first step in that direction) and in terms of application behavior.
For the later point, it's a bit more complicated because I realize that in a view service, we have no easy way detect when it becomes the active view service and when it's not anymore, `setNextViewServiceParameters` is called in that case, but this is not the right place to do such cleaning. This is a need we had before but we were able to workaround but given this issue, I think it's time to improve that. So I would suggest the following:

1. the base view service should define a boolean `active` attribute
1. when the app switch from one view service to the other (see the `activeViewService` attribute in the app), the `active` flag is updated
1. then the redirection attributes reset logic should be called when the create/edit view service becomes inactive.